inst/ms/aggreCAT.qmd

```diff
@@ -306,8 +344,7 @@ of estimates, and by prior statistical knowledge as measured in a quiz.
 
```

Comment:
L303: Each claim _outcome_ assumes binary values
L308: I don't think "an individual _i_ assesses the claim as being true or false through providing three probabilities" is the correct way of putting it. We're eliciting predictions of replication outcomes rather than the original being "true" or not.

Reply:
I just checked the Hanea2021 paper and that is how it is referred to there (minus wording), but I still think it isn't right. Could be for non-SCORE usage, but not for repliCATS.

Reply:
L336: "other than mean, median, and Bayesian" isn't quite right. Unless we're happy to lump all other mean-variants under "mean" here

Reply:
- L303 - I've inserted the missing text, and added a qualifier that we are talking about the outcome of a replication study
- L308 - thanks, that's clumsy wording, agree the estimates are probabilistic estimates of whether a replication outcome is TRUE / FALSE
- "other than mean, median, and Bayesian" -> "other than the **AverageWAggg()** and Bayesian approaches"